spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/configuration/TestDependencies.java

```diff
@@ -136,6 +137,11 @@ public TaskLauncher taskLauncher() {
 	}
 
 	@Bean
```

Comment:
We can remove the @Bean annotation and make this private because it is only used in the taskController bean.